SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -80,8 +80,10 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, QueuedIncomingRPCWaitTime(1.0f)
 	, QueuedIncomingRPCRetryTime(1.0f)
 	, QueuedOutgoingRPCRetryTime(1.0f)
-	, PositionUpdateFrequency(1.0f)
-	, PositionDistanceThreshold(100.0f) // 1m (100cm)
+	, MinimumTimeThreshold(1.0f)       // 1 second
```

Comment:
These should still include `PositionUpdate` - and we probably should include the time units although I see we're not consistent in that regard.
`PositionUpdateThresholdMinSeconds`
`PositionUpdateThresholdMinDistance`
`PositionUpdateThresholdMaxSeconds`
`PositionUpdateThresholdMaxDistance`